src/__tests__/NewProjectDialog.js

```diff
@@ -10,15 +9,15 @@ describe('Testing', () => {
     const customers = [
         {
             name: "Name one",
-            customerId: 1234
+            customer_id: 1234
```

Comment:
will we work in snail case in the whole frontend?

Reply:
only for endpoints that will talk to the ruby backend.